### PR TITLE
S2t1 api sign in endpoint

### PR DESCRIPTION
**Pull request by HadertS**
signin endpoint created along with supporting architecture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,90 @@
...
+
+**Method:**
+
+`POST`
+
+**Body Content**
+
+```
+{
+    "name":"Bob Gunderson"
```
Comment (alreuben) [inline README.md]: add comma after name bits

File: README.md [hunk 2]
```diff
@@ -1,2 +1,90 @@
 # year-cohort-projectName (eg: 22-jul-projectName)
-project template repo
+This is the back-end RESTful API for the final project for the 2023 Woodchucks team at io Academy.
+The front-end can be found [here](https://github.com/iO-Academy/23-jan-signin-fe).
+
+## Installation
+
+Requires a mongoDB database to be set up with the name `OfficeGuestBook` and collection named `GuestBook`
```
Comment (alreuben) [inline README.md:7]: add npm i express and nmp i mongodb

**Review by alreuben — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (HadertS) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (HadertS) [reply README.md:7]: Added "npm i" instead as this will install all dependancies.

**Pull request merged**